akka-http/src/main/scala/akka/http/scaladsl/server/directives/RouteDirectives.scala

```diff
@@ -38,15 +38,7 @@ trait RouteDirectives {
   def redirect(uri: Uri, redirectionType: Redirection): StandardRoute =
     StandardRoute {
       _. //# red-impl
-        complete {
-          HttpResponse(
-            status = redirectionType,
-            headers = headers.Location(uri) :: Nil,
-            entity = redirectionType.htmlTemplate match {
-              case ""       ⇒ HttpEntity.Empty
-              case template ⇒ HttpEntity(ContentTypes.`text/html(UTF-8)`, template format uri)
-            })
-        }
+        redirect(uri, redirectionType)
```

Comment:
gotta check where the `red-impl` is used and direct it towards the actual implementation of redirect now